src/commonMain/kotlin/it/krzeminski/snakeyaml/engine/kmp/nodes/Tag.kt

```diff
@@ -28,18 +27,10 @@ class Tag(
 
     val value: String = UriEncoder.encode(tag)
 
-    /**
-     * Create a global tag to dump the fully qualified class name
-     *
-     * @param clazz - the class to use the name
-     */
-    @Deprecated(
-        "Kotlin Reflection is not well supported on non-JVM platforms - manually specify the FQN instead",
-        ReplaceWith("Tag.forClass(clazz.qualifiedName)"),
```

Comment:
Is there a method `forClass`? I see only `forType`, was it a mistake? Still, this change should be fine since we're planning to bump the major version, only this `ReplaceWith` was a bit unfortunate.

Reply:
Oops yeah, looks like there was never a `forClass()` util function!